set-data/README.md

```diff
@@ -43,7 +43,7 @@ but you can customize how (or even if) it’s displayed:
 ```html
 <v-app data='{"c": 25}'>
 	<input v-model="c" type="number" /> ℃ =
-	<set-data name="f" :value="c * 1.8 + 32">{{ +f.toFixed(2) }} ℉</set-data>
+	<set-data name="f" :value="c * 1.8 + 32">{{ f.toFixed(2) }} ℉</set-data>
```

Comment:
I believe  `+f.toFixed(2)` is crucial here: `f` is of type `string`, and there is no `toFixed()` method on strings. So we need to convert it to a number either with `Number(f)`, or with `+f`.

Reply:
Ah, I see, thanks. For readability reasons, I think `Number(f)` or `parseFloat(f)` would be better.

Reply:
I agree that `Number(f)` is more readable (but definitely not `parseFloat()`)